interop-tests/native-ping-version.json

```diff
@@ -12,7 +12,6 @@
     "noise"
   ],
   "muxers": [
-    "mplex",
```

Comment:
Calling out this change explicitly. Supporting mplex requires some additional effort, given that the `libp2p::SwarmBuilder` only supports yamux. I don't think this additional effort is worth it. Any objections?

Additional context: https://github.com/libp2p/test-plans/pull/263

Reply:
Re-introduced mplex in 9ee251fbf.